vectordb_bench/backend/filter.py

```diff
@@ -0,0 +1,75 @@
+from enum import StrEnum
+
+from ..base import BaseModel
+
+
+class FilterType(StrEnum):
+    Int = "Int"  # test ">="
+    Label = "Label"  # test "=="
+    NonFilter = "NonFilter"
```

Comment:
Maybe rename to FilterOp, this has more readability and is better to maintain.
```python
class FilterOp(StrEnum):
    NumGE = "NumGE"
    StrEqual = "StrEqual"
    NonFilter = "NonFilter"
```

Reply:
nice~